<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de projets</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>monkey-face-cartoon.png</normaloff>monkey-face-cartoon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Projets</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nom du projet</string>
           </property>
           <property name="buddy">
            <cstring>cbProjectName</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbProjectName">
           <property name="toolTip">
            <string/>
           </property>
           <property name="statusTip">
            <string>Entrez ou selectionnez un projet</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Date de debut</string>
           </property>
           <property name="buddy">
            <cstring>deProjectBeginDate</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="deProjectBeginDate">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Date de fin</string>
           </property>
           <property name="buddy">
            <cstring>deProjectEndDate</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDateEdit" name="deProjectEndDate">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnCreateProject">
           <property name="text">
            <string>Creer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveProject">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sauver</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteProject">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpdateProject">
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="lstTasks"/>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Taches</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Nom de la tache</string>
           </property>
           <property name="buddy">
            <cstring>leTaskName</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leTaskName">
           <property name="placeholderText">
            <string>Entrez le nom de la tache</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Date de debut</string>
           </property>
           <property name="buddy">
            <cstring>deTaskBeginDate</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="deTaskBeginDate">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Date de fin</string>
           </property>
           <property name="buddy">
            <cstring>deTaskEndDate</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDateEdit" name="deTaskEndDate">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnCreateTask">
           <property name="text">
            <string>Creer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveTask">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sauver</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteTask">
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpdateTask">
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionOpenDB"/>
    <addaction name="actionNewDB"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu_Fichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenDB">
   <property name="text">
    <string>&amp;Ouvrir une db</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNewDB">
   <property name="text">
    <string>&amp;Nouvelle db</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCreateTask</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>createTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCreateProject</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>createProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSaveProject</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDeleteProject</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUpdateProject</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSaveTask</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDeleteTask</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUpdateTask</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNewDB</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newDB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenDB</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createProject()</slot>
  <slot>saveProject()</slot>
  <slot>deleteProject()</slot>
  <slot>updateProject()</slot>
  <slot>createTask()</slot>
  <slot>saveTask()</slot>
  <slot>deleteTask()</slot>
  <slot>updateTask()</slot>
  <slot>openDB()</slot>
  <slot>newDB()</slot>
 </slots>
</ui>
